README.md

```diff
@@ -222,13 +233,13 @@ before uploading PRs for review.
 
 ```bash
 # Format python code
-$ python -m black python
+$ python -m black ./python
```

Comment:
Since we re-installed black dependency in `build-and-test.yaml`, we may need to re-install it here as well.

Reply:
Discussed offline. We agree it is simpler to keep it as is and users should know how to install black if it is not find in the python library.